stormpath/stormpath-express-react-example
# Move /build files into /src

Cleans up the `/build` directory by moving it into `/src`.
#### How to verify
1. Checkout this branch.
2. Run the application with `$ npm start`.
3. Verify that it works the same as with the `master` branch.

Fixes #11
